docs/rootless.md

```diff
@@ -75,6 +75,17 @@ You can just use the upstream Docker client but you need to set the socket path
 $ docker -H unix://$XDG_RUNTIME_DIR/docker.sock run -d nginx
 ```
 
+### Expose Docker API socket via TCP
+
+To expose the Docker API socket via TCP, you need to launch `dockerd-rootless.sh` with `DOCKERD_ROOTLESS_ROOTLESSKIT_FLAGS="-p 0.0.0.0:2376:2376/tcp"`.
+
+```console
+$ DOCKERD_ROOTLESS_ROOTLESSKIT_FLAGS="-p 0.0.0.0:2376:2376/tcp" \
```

Comment:
@AkihiroSuda ok I finally got around testing this and it works, but for usability, I think it would be better to set a smarter default for `DOCKERD_ROOTLESS_ROOTLESSKIT_FLAGS` based on what's in `-H` or maybe that's too smart? (It would have to grep for `tcp://` in `-H` `--host` flags and `DOCKER_HOST` envvar).

Reply:
It would need to grep daemon.json as well, so too complex for shell script :P